Detectors/ITSMFT/MFT/tracking/src/TrackFitter.cxx

```diff
@@ -312,7 +295,7 @@ bool TrackFitter::computeCluster(TrackLTF& track, int cluster)
   const std::array<float, 2>& cov = {sigmaX2, sigmaY2};
 
   if (track.update(pos, cov)) {
-    if (mftTrackingParam.verbose) {
+    if (mVerbose) {
```

Comment:
Please do not use your own verbosity mechanism, but rely on the `Framework/Logger.h` /  `FairLogger.h` macros.